tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/service/mapper/ProgrammeMembershipMapper.java

```diff
@@ -86,23 +87,23 @@ public List<ProgrammeMembershipDTO> allEntityToDto(
 
   public List<ProgrammeMembershipDTO> programmeMembershipsToProgrammeMembershipDTOs(
       List<ProgrammeMembership> programmeMemberships) {
-    Map<ProgrammeMembershipDTO, ProgrammeMembershipDTO> listMap = Maps.newHashMap();
+    Map<UUID, ProgrammeMembershipDTO> listMap = Maps.newHashMap();
```

Comment:
Before splitting programme memberships from curriculum memberships, we didn't have a ID, so we had to use the hashcode method in ProgrammeMembershipDTO for hashmap keys.
Now, it would be better to use UUID instead.